functions/helpers.bash

```diff
@@ -87,6 +87,8 @@ update_git_repo() {
 get_public_ip() {
   local pubIP
 
+  if ! [[ -x $(command -v dig) ]]; then return 1; fi
```

Comment:
are we checking return value in the calling routine ? just a thought

Reply:
Nope just the string if I understand bash logic correctly.

Reply:
ok but yes when it fails the return string _should_ be empty and that the calling routines check against so it should do.